src/Development/IDE/Core/Shake.hs

```diff
@@ -62,8 +62,9 @@ module Development.IDE.Core.Shake(
     DelayedAction, mkDelayedAction,
     IdeAction(..), runIdeAction,
     mkUpdater,
+    LinkEnvsCache,
     -- Exposed for testing.
-    Q(..),
+    Q(..),    
```

Comment:
trimming whitespace needed

Reply:
the project has a `.editorconfg` that enforces that, do you have installed the needed plugin in your editor to honour it?
It saves a lot of time fixing those little things automatically.

Reply:
thanks for the hint